terraform/application/config/production_app_env.yml

```diff
@@ -0,0 +1 @@
+---
```

Comment:
we don't really seem to be using these files - i wonder whether we pull the 'non-secret' secrets out of the keyvaults and put them here. Things like `HMRC_API_BANK_VALIDATION_ENABLED`

or just move the couple that we do have in test/review e.g. `BYPASS_DFE_SIGN_IN` into the keyvault? cc @slorek

Reply:
Yes please move the non secret values to yml. It makes the deployment faster, you can track the values more easily, and change them via PR in a controlled way.
But you may do that in another PR.

Reply:
addressing in new PR 👍